src/EpiContentUsage/Frontend/Lib/hooks/useFilteredTableData.ts

```diff
@@ -79,6 +83,16 @@ export function useFilteredTableData<TableDataType>({
   const [searchParams, setSearchParams] = useSearchParams();
   const location = useLocation();
 
+  const debouncedSetSearchParams = useDebounce<
+    (prev: URLSearchParams) => URLSearchParamsInit
```

Comment:
Can't be just `(prev: URLSearchParams) => URLSearchParams`?

Reply:
Yes. You're right. Done